File: Domain/Automobil.cs
using System;
using System.Collections.Generic;

namespace Domain
{
    public class Automobil
    {
        public Guid Id { get; set; }
        public String model { get; set; }
        public String cena { get; set; }
        public String slika { get; set; }
        public ICollection<SveOAutomobilu> SveOAutomobilima { get; set; }
    }
}

File: Client/Javascript/Automobil/automobil.js
export class Automobil
{
    constructor(id, model, cena, slika, SveOAutomobilima)
    {
        this.id = id;
        this.model = model;
        this.cena = cena;
        this.slika = slika;
        this.kwks = [];
        this.pogoni=[];
        this.karoserije=[];
        this.kubikaze=[];
        this.oprema=[];
        SveOAutomobilima.forEach(SveOAutomobilu => {
            this.karoserije.push(SveOAutomobilu.vrsteKaroserija);
            this.kubikaze.push(SveOAutomobilu.vrsteKubikaza);
            this.kwks.push(SveOAutomobilu.vrsteKwks);
            this.pogoni.push(SveOAutomobilu.vrstaPogona);
            this.oprema.push(SveOAutomobilu.vrstaOpreme);
        });
        /*this.karoserija = karoserija;
        this.kubikaza = kubikaza;
        
        this.pogon = pogon;
        this.oprema = oprema;;*/
    }
}